Essentials/src/config.yml

```diff
@@ -484,6 +484,19 @@ send-fly-enable-on-join: true
 # Give someone permission to teleport to a world with essentials.time.world.<worldname>.
 world-time-permissions: false
 
+# Specify cooldown for both Essentials commands and external commands as well.
+# All commands do not start with a Forward Slash (/). Instead of /msg, write msg
+#
+# Wildcards are supported. E.g.
+# - '*i*': 50
+# adds a 50 second cooldown to all commands that include the letter i
+#command-cooldowns:
+#  feed: 100 # 100 second cooldown on /feed command
```

Comment:
No because it just means they will have less to uncomment if they do want to turn it on. Otherwise, it would still be off since there aren't any actual cooldowns there.